lib/tasks/templates.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Mix.Tasks.Addict.Gen.Templates do
+  use Mix.Task
+  import Mix.Generator
+  alias Addict.Utils
+
+  @shortdoc "Generate templates and view for Addict"
+  @moduledoc """
+  Generates an Addict view and templates
+      mix addict.gen.templates
```

Comment:
We prefix this mix task with `addict` to keep from any namespace collisions. I'd prefer to name it by what it does (create templates) to keep from confusion if possible but I also have no problems with `screens` if you feel strongly. :smile: